analogwp-templates.php

```diff
@@ -126,19 +126,44 @@ function analog_fail_load() {
 	if ( ! function_exists( 'get_plugins' ) ) {
 		require_once ABSPATH . 'wp-admin/includes/plugin.php';
 	}
-	$file_path         = 'elementor/elementor.php';
-	$installed_plugins = get_plugins();
-	$elementor         = isset( $installed_plugins[ $file_path ] );
+	$file_path = 'elementor/elementor.php';
 
-	if ( $elementor ) {
+	if ( version_compare( get_bloginfo( 'version' ), '5.5a', 'gt' ) ) {
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/plugins/elementor/elementor' );
+		$response = rest_do_request( $request );
+
+		if ( $response->is_error() ) {
+			wp_die( '<p>' . esc_html__( 'An error occurred while checking Elementor is Installed/Activated', 'ang' ) . '</p>' );
```

Comment:
Don't use `wp_die`, if a user reaches here, they'll have no option to return back.

Simply assigned that to `$message` for error display, or just return.

Reply:
Handled error message display in a user-friendly way.